From 8a76d75392291c80a6b303133b95a31d98fd2f5c Mon Sep 17 00:00:00 2001
From: Andrew Keller <andrew@wimpyprogrammer.com>
Date: Wed, 18 Feb 2015 22:45:15 -0500
Subject: [PATCH] Fixing type annotations in domain.js

Fixing the syntax of the type annotations for variable number, any type callbacks to avoid warnings from the Closure Compiler.

Upstream-Commit: https://github.com/dcodeIO/node.js-closure-compiler-externs/commit/8a76d75392291c80a6b303133b95a31d98fd2f5c
---
 domain.js | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/domain.js b/domain.js
index 3349561..6136f91 100644
--- a/domain.js
+++ b/domain.js
@@ -64,14 +64,14 @@ domain.Domain.prototype.add = function(emitter) {};
 domain.Domain.prototype.remove = function(emitter) {};
 
 /**
- * @param {function(...[*])} callback
- * @return {function(...[*])}
+ * @param {function(...*)} callback
+ * @return {function(...*)}
  */
 domain.Domain.prototype.bind = function(callback) {};
 
 /**
- * @param {function(...[*])} callback
- * @return {function(...[*])}
+ * @param {function(...*)} callback
+ * @return {function(...*)}
  */
 domain.Domain.prototype.intercept = function(callback) {};
 
